tests/unit/aiplatform/test_hyperparameter_tuning_job.py

```diff
@@ -377,6 +403,12 @@ def test_create_hyperparameter_tuning_job(
             base_output_dir=test_custom_job._TEST_BASE_OUTPUT_DIR,
         )
 
+        conditional_parameter_decay = hpt.DoubleParameterSpec(
```

Comment:
These two are defined multiple times. We can declare them globally. 
E.g, `_TEST_CONDITIONAL_PARAMETER_DECAY = hpt.DoubleParameterSpec(...)`

Reply:
added global definition